src/index.html

```diff
@@ -0,0 +1,2 @@
+<p>Hello world</p>
```

Comment:
This is a fine change, though it doesn't seem quite necessary for this Pull Request, as this PR is a more 'behind the scenes' change to the tetris pieces.

As there's a lot of devs working on the same files, we should be vigilant about making sure we don't change too many files at once. If merged, then all other branches would need to account for this change. It's not the end of the world or anything, Git & GitHub will let you sort it out, but it's something we should try to avoid to help minimise the risk of scary merge conflicts!

In general, if you'd like to make a change that's not to do with the task at hand (placing a piece on the grid), it should be a separate branch & pull request, so we should probably remove this change (plenty of other branches touch index.html right now!)